src/geodesic.rs

```diff
@@ -914,11 +917,6 @@ impl Geodesic {
         if !arcmode {
             outmask = outmask | caps::DISTANCE_IN;
         };
-        let lon1 = if outmask & caps::LONG_UNROLL != 0 {
```

Comment:
I guess this is redundant here because Karney had added duplicate behavior in `GeodesicLine`... right?

Reply:
Exactly. This is another case where the tip-off was that the C++ code didn't bother with it here. This is a case where the old difference almost certainly did no harm other than slight performance overhead, but I'm just sort of gradually shifting things toward the C++ approach.